src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ReaderServiceImplTest {
+    private static final String EMPTY_FILE_NAME = "src/test/resources/test1.csv";
+    private static final String FIRST_FILE = "src/test/resources/FruitShop.csv";
+    private static final String PATH_TO_NON_EXISTING_FILE = "src/Main/java/resources/cars.csv";
```

Comment:
better not use capital letters in file path
```suggestion
    private static final String PATH_TO_NON_EXISTING_FILE = "src/main/java/resources/cars.csv";
```